asmcli/commands/install.sh

```diff
@@ -83,21 +83,20 @@ install_private_ca() {
   # This sets up IAM privileges for the project to be able to access GCP CAS.
   # If modify_gcp_component permissions are not granted, it is assumed that the
   # user has taken care of this, else Istio setup will fail
-  local ISTIOD_SERVICE_ACCOUNT
-  ISTIOD_SERVICE_ACCOUNT="istiod-${REVISION_LABEL}"
-  if [[ "${_CI_NO_REVISION}" -eq 1 ]]; then
-    ISTIOD_SERVICE_ACCOUNT="istiod"
-  fi
-  local WORKLOAD_IDENTITY; WORKLOAD_IDENTITY="${WORKLOAD_POOL}[istio-system/${ISTIOD_SERVICE_ACCOUNT}]"
-  local NAME; NAME=$(echo "${CA_NAME}" | cut -f6 -d/)
+  local CA_NAME; CA_NAME="$(context_get-option "CA_NAME")"
+
+  local WORKLOAD_IDENTITY; WORKLOAD_IDENTITY="${WORKLOAD_POOL}[-/-]"
```

Comment:
Unsure if WORKLOAD_POOL needs to be retrieved from context. It doesn't appear to be set anywhere

Reply:
Does everything work correctly when you test locally?

Reply:
yes